tests/TestCase/Auth/DigestAuthenticateTest.php

```diff
@@ -61,12 +61,14 @@ public function setUp()
     {
         parent::setUp();
 
+        $salt = 'foo.bar';
+        Security::setSalt($salt);
```

Comment:
This would need to be reset although I'm not sure why it needs to be changed for this test.

Reply:
That is a valid question. But we should check that on 4.x first, and if we find the reason to do so, I can update this PR.

Reply:
You were right, in 4.next latest this is also not there anymore, so I cleaned this up now too.